kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/internal/PodOperationsImpl.java

```diff
@@ -335,22 +335,5 @@ public ExecWatch exec(String... command) {
     public BytesLimitTerminateTimeTailPrettyLoggable<String, LogWatch> limitBytes(int limitBytes) {
         return new PodOperationsImpl(client, getConfig(), apiVersion, namespace, name, isCascading(), getItem(), getResourceVersion(), isReloadingFromServer(), getGracePeriodSeconds(), getLabels(), getLabelsNot(), getLabelsIn(), getLabelsNotIn(), getFields(), containerId, in, inPipe, out, outPipe, err, errPipe, withTTY, withTerminatedStatus, withTimestamps, sinceTimestamp, sinceSeconds, withTailingLines, withPrettyOutput, execListener, limitBytes);
     }
-
-  @Override
-  public Pod waitUntilReady(long amount, TimeUnit timeUnit) throws InterruptedException {
-    Pod pod = get();
-    if (pod == null) {
-      throw new IllegalArgumentException("Pod with name:[" + name + "] in namespace:[" + namespace + "] not found!");
-    }
-
-    if (Readiness.isReady(pod)) {
-      return pod;
-    }
-
-    ReadinessWatcher<Pod> watcher = new ReadinessWatcher<>(pod);
-    try (Watch watch = watch(watcher)) {
-      return watcher.await(amount, timeUnit);
-    }
-  }
```

Comment:
Just thinking -- not sure why `Pod.waitUntilReady` was different than `<everything else>.waitUntilReady`, but the common implementation seems to work just fine.

Reply:
see my comment above.